02-ca-certificates.md

```diff
@@ -24,11 +24,11 @@ Following the steps below you will result the certificate needed for Azure Appli
    :warning: We called bicycle3 and bicycle4 each subdomain, but the DNS values could be not available. In that case, you can change the following values.
 
    ```bash
-   CLUSTER_SUBDOMAIN_03=bicycle3
-   CLUSTER_SUBDOMAIN_04=bicycle4
+   export CLUSTER_SUBDOMAIN_03=bicycle3
```

Comment:
we had a cumulated commit with all missing exports from [dade9](https://github.com/mspnp/aks-secure-baseline/pull/155/commits/dade95a1f0f54f6be8ed7f9ab84562c1262d69e8).

But it seems in some other [merge](https://github.com/mspnp/aks-secure-baseline/commit/eb4eb480185741e1ab46d2a4580736e5ef1beb64) we could've retired some. 

Anybody knows what is the expected format/style? Personally I'd say, all-or-nothing but I'm open and would love to hear more about this.

cc/ @ckittel

Reply:
remove all the exports for preventing users from scrolling. We are not using the benefits from exporting env vars.

Reply:
done | addressed from:
[4b426](https://github.com/mspnp/aks-secure-baseline/pull/155/commits/4b426e834768a086d5bc5c880e991023c2c1f90e)
[55d94](https://github.com/mspnp/aks-secure-baseline/pull/155/commits/55d942653b604a72e380ff2e8728d92148ed5aca)